website/docs/docs/cloud/visual-editor-interface.md

```diff
@@ -2,7 +2,7 @@
 title: "Navigate the interface" 
 id: visual-editor-interface      
 sidebar_label: "Navigate the interface" 
-description: "The visual editor interface contains an operator toolbar, operators, and a canvas to help you create dbt models through a seamless drag-and-drop experience in dbt Cloud." 
+description: "The visual editor interface contains an operator toolbar, operators, and a canvas to help you access and transform data through a seamless drag-and-drop experience in dbt Cloud." 
```

Comment:
@rcg798 I think since we stress that this is VEs ultimate end goal is a model that we keep that language in the description to optimize SEO. How does this look to you:
```suggestion
description: "The Visual Editor interface contains an operator toolbar, operators, and a canvas to help you access and transform data through a seamless drag-and-drop model creation experience in dbt Cloud." 
```